internal/backend/sessions/sessionworkflows/workflow.go

```diff
@@ -111,7 +111,7 @@ func runWorkflow(
 
 func (w *sessionWorkflow) cleanupSignals(ctx workflow.Context) {
 	goCtx := temporalclient.NewWorkflowContextAsGOContext(ctx)
-	for signalID := range w.signals {
+	for signalID := range w.lastReadEventSeqForSignal {
```

Comment:
I like the rename for readability, but it makes this part is now less readable

Reply:
that's the most meaningful name i could come up with.